apps/web/src/pages/quick-start/components/demo-flow/nodes/NodeStep.tsx

```diff
@@ -161,12 +162,10 @@ function useCounter() {
 
 export function Description({ label, description, url }: { label: string; description: string; url?: string }) {
   const segment = useSegment();
+  const capitalizedLabel = label.charAt(0).toUpperCase() + label.slice(1);
 
   function handleOnClick() {
-    /*
-     * todo add ('label' will probably be needed here)
-     * segment.track(OnBoardingAnalyticsEnum);
-     */
+    segment.track(`${OnBoardingAnalyticsEnum.BUILD_WORKFLOW_NODE_POPOVER_LEARN_MORE_CLICK} On ${capitalizedLabel}`);
```

Comment:
Here i added the channel name under the same event (with no param), i am not sure it is the best way.. 

In my opinion, we should open a discussion about what should be the format of our events, knowing that the amount will grow in the future it will be best if we will have some sort of convention on how we should print them, so all the parties will be satisfied.

Reply:
I will raise it on discord active projects for discussion.